modules/oe_translation_local/modules/oe_translation_multivalue/tests/src/Functional/MultivalueTranslationsTest.php

```diff
@@ -405,6 +265,70 @@ public function testMultivalueFieldTranslationsApi(): void {
     }
   }
 
+  /**
+   * Tests the multivalue field delta removal.
+   *
+   * Tests that when in the original, we remove one of the field deltas, this
+   * gets removed also when translating. Just as it does if it's a regular
+   * single value text field.
+   */
+  public function testMultivalueFieldValueRemoval(): void {
+    $node = $this->createNodeWithMultivalueFields();
+
+    // Remove the delta 0 on all the fields.
+    $paragraph = $node->get('field_paragraphs')->entity;
+    $paragraph->setNewRevision();
+    $values = $paragraph->get('field_textfield_paragraph')->getValue();
+    unset($values[0]);
+    $paragraph->set('field_textfield_paragraph', $values);
+    $paragraph->save();
+    foreach ($this->getMultivalueFieldNames() as $field_name) {
+      $values = $node->get($field_name)->getValue();
+      unset($values[0]);
+      $node->set($field_name, $values);
+      $node->set('field_paragraphs', [
+        'target_id' => $paragraph->id(),
+        'target_revision_id' => $paragraph->getRevisionId(),
+      ]);
+      $node->setNewRevision();
```

Comment:
I didn't get why we are updating same field value in each iteration. 
```suggestion
    $paragraph->save();
    $node->set('field_paragraphs', [
      'target_id' => $paragraph->id(),
      'target_revision_id' => $paragraph->getRevisionId(),
    ]);
    foreach ($this->getMultivalueFieldNames() as $field_name) {
      $values = $node->get($field_name)->getValue();
      unset($values[0]);
      $node->set($field_name, $values);
      $node->setNewRevision();
```